### ISSUE TITLE
Not getting expected update on docker image

### ISSUE THREAD

**Issue by cblecker** [dependabot/dependabot-core#7562]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Package ecosystem

Docker

### Manifest location and content before the Dependabot update

https://github.com/cblecker/ubi-dependabot-test/blob/main/build/Dockerfile

### dependabot.yml content

https://github.com/cblecker/ubi-dependabot-test/blob/main/.github/dependabot.yml

### Updated dependency

Current version: registry.access.redhat.com/ubi8/ubi-minimal:8.8-860
Expected update: registry.access.redhat.com/ubi8/ubi-minimal:8.8-1014

### What you expected to see, versus what you actually saw

I would expect to see a dependabot update from 8.8-860 to 8.8-1014, but for some reason dependabot still sees 8.8-860 as the latest version and I'm not entirely sure why.

```
updater | 2023/07/14 16:01:53 INFO <job_694938276> Checking all dependencies for version updates...
updater | 2023/07/14 16:01:53 INFO <job_694938276> Checking if ubi8/ubi-minimal 8.8-860 needs updating
  proxy | 2023/07/14 16:01:53 [014] GET https://registry.access.redhat.com:443/v2/ubi8/ubi-minimal/tags/list
  proxy | 2023/07/14 16:01:53 [014] 200 https://registry.access.redhat.com:443/v2/ubi8/ubi-minimal/tags/list
updater | 2023/07/14 16:01:53 INFO <job_694938276> Latest version is 8.8-860
updater | 2023/07/14 16:01:53 INFO <job_694938276> No update needed for ubi8/ubi-minimal 8.8-860
```

### Smallest manifest that reproduces the issue

```
FROM registry.access.redhat.com/ubi8/ubi-minimal:8.8-860
```

**Comment by NautiluX** [comment]
This issue should be fixed with PR #7578

**Comment by NautiluX** [comment]
@jurre can you reopen this issue?
